component/telemetry.go

```diff
@@ -22,10 +22,18 @@ type TelemetrySettings struct {
 	TracerProvider trace.TracerProvider
 
 	// MeterProvider that the factory can pass to other instrumented third-party libraries.
+	//
+	// Deprecated [v0.108.0]: use LeveledMeterProvider instead.
 	MeterProvider metric.MeterProvider
 
+	// LeveledMeterProvider returns a MeterProvider for a Level that the factory can
+	// pass to other instrumented third-party libraries.
+	LeveledMeterProvider func(level configtelemetry.Level) metric.MeterProvider
```

Comment:
Does this dependency mean we can't mark `component` stable until `configtelemetry` is stable?

Reply:
Yes, but this dependency was already there, right?

Reply:
You're right